vyxal/vyxal.py

```diff
@@ -0,0 +1,21 @@
+"""
+File: vyxal.py
+Description: This is the main file for the project - the transpilation
+of Vyxal programs actually happens here, and this is what gets executed
+offline.
+"""
+
+from vyxal.context import Context
+from vyxal.transpile import transpile
+
+ctx = Context()
+
+if __name__ == "__main__":
+    # I'm allowed to have this here this time. Frick you if you say I
+    # can't.
+
+    while True:
```

Comment:
is every line a separate vyxal program, or do we have plans of a repl where all commands effect the global stack?

Reply:
Every line is a different vyxal program (at least that's how I intend it)